<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans Mono Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: transparent;
	padding: 0;
	margin: 0;
	border: none;
	color: #fff;
}
QPushButton{
	background-color: rgba(14, 14, 22, 100);
	border: 2px solid rgb(24, 24, 36);
	border-radius: 5px;
}
#centralwidget{
	background-color: rgb(24, 24, 36);
	text-align: left;
}
#footer{	
	background-color: rgb(9, 5, 13);
	padding: 10px;
}
#messages_widget,
#menu_widget,
#right_menu,
#search_widget,
#notifications_widget,
#bottom_menu,
#login_widget,
#header_search_widget{	
	background-color: rgb(9, 5, 13);
	padding: 10px;
	border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="main_body">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="main_body_contents">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton, QLineEdit{
	padding: 5px;
}
QLineEdit{
	border-bottom:  2px solid rgb(24, 24, 36);
}
background-color: rgb(114, 159, 207);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_14">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCustomSlideMenu" name="right_menu" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_21">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_13">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>300</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_20">
                   <item>
                    <widget class="QCustomSlideMenu" name="section1" native="true">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>250</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>200</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_15">
                      <item>
                       <widget class="QFrame" name="frame_8">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>130</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>350</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="frameShape">
                         <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                         <enum>QFrame::Raised</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_3">
                         <item>
                          <widget class="QGroupBox" name="groupBox_4">
                           <property name="minimumSize">
                            <size>
                             <width>0</width>
                             <height>100</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>300</width>
                             <height>70</height>
                            </size>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);</string>
                           </property>
                           <property name="title">
                            <string>Server</string>
                           </property>
                           <layout class="QVBoxLayout" name="verticalLayout_6">
                            <item>
                             <widget class="QFrame" name="frame_10">
                              <property name="frameShape">
                               <enum>QFrame::StyledPanel</enum>
                              </property>
                              <property name="frameShadow">
                               <enum>QFrame::Raised</enum>
                              </property>
                              <layout class="QHBoxLayout" name="horizontalLayout_17">
                               <item>
                                <widget class="QPushButton" name="btn_try">
                                 <property name="minimumSize">
                                  <size>
                                   <width>60</width>
                                   <height>25</height>
                                  </size>
                                 </property>
                                 <property name="maximumSize">
                                  <size>
                                   <width>60</width>
                                   <height>30</height>
                                  </size>
                                 </property>
                                 <property name="styleSheet">
                                  <string notr="true">border-color: rgb(49, 168, 255);</string>
                                 </property>
                                 <property name="text">
                                  <string>Launch</string>
                                 </property>
                                </widget>
                               </item>
                               <item>
                                <widget class="QProgressBar" name="progressserv">
                                 <property name="minimumSize">
                                  <size>
                                   <width>0</width>
                                   <height>25</height>
                                  </size>
                                 </property>
                                 <property name="value">
                                  <number>0</number>
                                 </property>
                                </widget>
                               </item>
                              </layout>
                             </widget>
                            </item>
                            <item>
                             <widget class="QFrame" name="frame_16">
                              <property name="frameShape">
                               <enum>QFrame::StyledPanel</enum>
                              </property>
                              <property name="frameShadow">
                               <enum>QFrame::Raised</enum>
                              </property>
                              <layout class="QHBoxLayout" name="horizontalLayout_16">
                               <item>
                                <widget class="QPushButton" name="detener_servidor">
                                 <property name="minimumSize">
                                  <size>
                                   <width>60</width>
                                   <height>25</height>
                                  </size>
                                 </property>
                                 <property name="maximumSize">
                                  <size>
                                   <width>60</width>
                                   <height>30</height>
                                  </size>
                                 </property>
                                 <property name="styleSheet">
                                  <string notr="true">border-color: rgb(49, 168, 255);</string>
                                 </property>
                                 <property name="text">
                                  <string>Stop </string>
                                 </property>
                                </widget>
                               </item>
                               <item>
                                <widget class="QProgressBar" name="progress_cerrar">
                                 <property name="minimumSize">
                                  <size>
                                   <width>0</width>
                                   <height>25</height>
                                  </size>
                                 </property>
                                 <property name="value">
                                  <number>0</number>
                                 </property>
                                </widget>
                               </item>
                              </layout>
                             </widget>
                            </item>
                           </layout>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomSlideMenu</class>
   <extends>QWidget</extends>
   <header location="global">Custom_Widgets.Widgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
